app.rb

```diff
@@ -1,8 +1,16 @@
+# This file contains Google Cloud Functions
+# - cartoons_list_update:
+#   Fetch list of cartoon update, and check that each need to update to db or not
+#   Trigger by Google Cloud Scheduler, cron job on every hour
+# - cartoon_update
+#   Check on every cartoon update that it needs to notify to any subscriber or not
+#   Trigger by Google Firestore update
+
 require 'functions_framework'
 
-PROJECT_ID = 'niceoppai-notifier'.freeze
-CARTOON_LIST_COLLECTION = 'cartoons'.freeze
-ACCOUNT_LIST_COLLECTION = 'accounts'.freeze
+PROJECT_ID = 'niceoppai-notifier'
+CARTOON_LIST_COLLECTION = 'cartoons'
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[Correctable] Style/MutableConstant: Freeze mutable objects assigned to constants.